py3dtiles/b3dm.py

```diff
@@ -112,16 +116,20 @@ def sync(self, body):
         self.ft_json_byte_length = 0
         self.ft_bin_byte_length = 0
 
+        if body.feature_table is not None:
+            fth_arr = body.feature_table.to_array()
+            # ftb_arr = body.feature_table.body.to_array()
```

Comment:
Is there a point in keeping the commented line? If it illustrate a futur needs, maybe we can explicit it in a comment ?